client/src/lib/error-telemetry.js

```diff
@@ -9,22 +9,47 @@ const Tail   = require('tail').Tail;
 const fs     = require('fs');
 const logger = require('winston');
 
-const TARGET = '/tmp/test';
+/**
+ * Default behaviour for the output where to send data to when the watched logging file has
+ * a modification detected.
+ */
+function callErrorTelemetryService(app, text) {
```

Comment:
It's unclear to me why this is living outside of the `ErrorTelemetry` class? It seems perfectly reasonable to live within it.

Reply:
AIUI, this means this method would be exported, and usable from outside, which I do not really want.

Reply:
I'm not sure the perceived benefit of "private methods" is worth it, this makes this function untestable.

There are a lot of things in Registration for example which in Java terms would be considered "protected" but I've not bothered spending any time trying to hide functions from ourselves. IMO if the function is conceptually private, then commenting in the API docs appropriately should be fine.